source/types.d.ts

```diff
@@ -0,0 +1,9 @@
+import type { User } from './models';
+
+declare global {
+	namespace Express {
+		interface Locals {
+			user: User | null;
```

Comment:
This can be undefined as well, add a disclaimer in source/index.ts about the user being undefined before the middleware for auth is recognized.

Reply:
Done, see [0d7687c](https://github.com/Scarpy019/TTII/pull/1/commits/0d7687c9603aca088f50d92e9f98638df130eca1).